### DISCUSSION TITLE
"Unable to resolve host" messages in mastercontainer logs

### DISCUSSION THREAD

**Discussion by MrSuttonmann** [nextcloud/all-in-one#1666] [Questions] [answered]
Hi, starting up AIO for the first time (in reverse proxy mode, with docker compose), and I notice in the mastercontainer logs a lot of "unable to resolve host" messages, like so:

`nextcloud-aio-mastercontainer  | [Wed Jan 04 10:51:43.718575 2023] [php:notice] [pid 218] [client 127.0.0.1:33402] Could not get digest of container nextcloud/all-in-one:latest cURL error 6: Could not resolve host: registry-1.docker.io (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://registry-1.docker.io/v2/nextcloud/all-in-one/manifests/latest, referer: https://MYDOMAIN:8181/login`

AIO does seem to be downloading and  starting up the containers properly however.

Are these messages important or are they okay to ignore? For reference I'm using PiHole as a DNS server which is on the same host in another docker container. No other docker containers seem to have any issues resolving domains however, and the domains attempting to be resolved are not blocked by PiHole.

**Comment by szaimen** [comment, marked as answer]
Hi, can you try to set the dns server to your pi-hole for the docker daemon globally via /etc/docker/daemon.json, reboot your server and check if that makes it work?
See https://dockerlabs.collabnix.com/intermediate/networking/Configuring_DNS.html

**Comment by MrSuttonmann** [comment]
Thanks, that's solved it!

**Reply by szaimen** [reply]
cool, great!

**Comment by szaimen** [comment]
Moving to discussion so that others can find this

**Reply by samuelborn** [reply]
Thank you for moving. Also fixed my issue.
